### PR TITLE
Fix volumeAvailableCapacityForImportantUsage error

### PR DESCRIPTION
**Pull request by kingiol**
when use Device.volumeAvailableCapacityForImportantUsage cause error:

> [client] Failed to get available space for volume /: Error Domain=CacheDeleteErrorDomain Code=10 "(null)" UserInfo={Volume=/}

### CODE REVIEW

**Comment by dennisweissmann** [comment]
Thank you for this PR, I don’t know who originally contributed that (I still o my have a mobile right now), I think the change makes sense though!

Thanks also for reviewing the PR @Zandor :) I can only second you review!

**Comment by kingiol** [comment]
Thxs @KarlZeo , I also the gyb file.

File: Source/Device.generated.swift [hunk 1]
```diff
@@ -1100,7 +1100,7 @@ extension Device {
   /// Return the root url
   ///
   /// - returns: the "/" url
```
Comment (Zandor300) [inline Source/Device.generated.swift]: Please compile the project in Xcode so the change to this line in your gyb file propagates to the `.generated.swift` file.

Comment (kingiol) [reply Source/Device.generated.swift]: Already done.

**Pull request merged**